phys/module_sf_sfclay.F

```diff
@@ -521,11 +532,8 @@ SUBROUTINE SFCLAY1D(J,UX,VX,T1D,QV1D,P1D,dz8w1d,                &
         VCONV = SQRT(DTHVM)
         endif
 ! Mahrt and Sun low-res correction
-        if(present(dxCell)) then
-           vsgd = 0.32 * (max(dxCell(i)/5000.-1.,0.))**.33
-        else
-           VSGD = 0.32 * (max(dx/5000.-1.,0.))**.33
-        endif
+        VSGD = 0.32 * (max(dx(i)/5000.-1.,0.))**.33
+!MPAS specific end.
```

Comment:
@kkeene44 The comment can be removed now.

Reply:
@weiwangncar 
I removed and updated the code.